src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+
+  testData.forEach((tooltipId) => {
+    test(`should check tooltipId as ${tooltipId}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipId={tooltipId} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toHaveId(tooltipId);
+    });
+  });
+
+  (["top", "bottom", "left", "right"] as any[]).forEach((tooltipPosition) => {
+    test(`should render with tooltip positioned ${tooltipPosition}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest tooltipPosition={tooltipPosition} isFocused>
+          {`This tooltip is positioned ${tooltipPosition}`}
+        </HelpComponentTest>
+      );
+      await expect(helpComponent(page)).toBeVisible();
+
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toBeVisible();
+      await expect(tooltip).toHaveAttribute("data-placement", tooltipPosition);
+    });
+  });
+
+  (["error", "add", "minus", "settings"] as any[]).forEach((iconType) => {
+    test(`should render with iconType prop passed as ${iconType}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<HelpComponentTest isFocused type={iconType} />);
+
+      await expect(icon(page)).toBeVisible();
+      await expect(icon(page)).toHaveAttribute("type", iconType);
+    });
+  });
+});
+
+test.describe("Accessibility tests for Help component", () => {
+  testData.forEach((ariaLabel) => {
+    test(`should check ariaLabel as ${ariaLabel}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest ariaLabel={ariaLabel} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((className) => {
+    test(`should check className as ${className}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest className={className} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  ["https://carbon.sage.com", "google", "https://www.google.com"].forEach(
+    (hrefLink) => {
+      test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+        await checkAccessibility(page);
+      });
+    }
+  );
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+      await checkAccessibility(page);
```

Comment:
**suggestion: (non-blocking)** as above, it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.